src/libecalc/application/graph_result.py

```diff
@@ -793,17 +789,14 @@ def get_asset_result(self) -> libecalc.dto.result.EcalcModelResult:
                                 energy_usage=TimeSeriesRate.from_timeseries_stream_day_rate(
                                     model.energy_usage, regularity=regularity
                                 ),
-                                energy_usage_unit=model.energy_usage_unit,
                                 requested_inlet_pressure=requested_inlet_pressure,
                                 requested_outlet_pressure=requested_outlet_pressure,
                                 rate=rate,
                                 maximum_rate=maximum_rate,
                                 stage_results=model_stage_results,
                                 failure_status=model.failure_status,
                                 timesteps=model.timesteps,
-                                is_valid=TimeSeriesBoolean(
-                                    timesteps=model.timesteps, values=model.is_valid, unit=Unit.NONE
-                                ),
+                                is_valid=model.is_valid,
```

Comment:
Is it not supposed to a TimeSeriesBoolean anymore here?

Reply:
Yes, but it already is in the model result